internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/DaVinciClientTest.java

```diff
@@ -1109,10 +1110,10 @@ public void testBlobP2PTransferAmongDVC() throws Exception {
   }
 
   /**
-   * Test to verify the snapshot generation for plain table.
+   * Test to verify the snapshot generation
    */
-  @Test(timeOut = 2 * TEST_TIMEOUT)
-  public void testDVCSnapshotGenerationForPT() throws Exception {
+  @Test(dataProviderClass = DataProviderUtils.class, dataProvider = "True-and-False", timeOut = 2 * TEST_TIMEOUT)
+  public void testDVCSnapshotGeneration(boolean useDiskStorage) throws Exception {
```

Comment:
nit: Nice! Do you think we can delete `testBlobP2PTransferAmongDVC` or rename one to another? Both integ tests have many similarity so I'd like to have only one of them. If you'd like to do it later, that's okay too.

Reply:
Lets keep testBlobP2PTransferAmongDVC, as it contains some logic related to transfer from DVC1 to DVC2, which should can be reused when we test transfer stage.